src/style.css

```diff
@@ -94,3 +94,61 @@ header {
   color: #f04;
   font-size: 1rem;
 }
+
+footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.footer-content {
+  width: 80%;
+  margin: auto;
+  padding: 20px 0;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.footer-content img {
+  width: 50px;
+}
+
+.footer-content section {
+  width: 30%;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.footer-content section:first-child {
+  align-items: flex-start;
+}
+
+.footer-content section h2 {
+  font-size: 18px;
+  color: #f04;
+  font-family: 'Bebas Neue', cursive;
+}
+
+.footer-content section ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+}
+
+.footer-content section:first-child ul {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
[OPTIONAL] In my opinion I feel like it would be better to use `justify-content: flex-start` and add `gap: 1rem`
The result would look like this
<img width="548" alt="Screenshot 2021-12-22 at 5 33 06 pm" src="https://user-images.githubusercontent.com/45717589/147087020-b310a2b5-39c3-4a7e-854f-0e2a5d710589.png">

Reply:
ok i'll apply it rigth away